readme/169-qr-code-specification-1.2.2.md

```
@@ -0,0 +1,430 @@
+    # Voice template
+    - 0: 03CBA(...)0378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorZ # Biometric data issuer
+```
+
+#### 3.2.1 Steps for Claim 169 Compliant QR Code Generation
+
+* **Prepare Identity Data**: Start with the sample JSON identity data provided for conversion into Claim 169 format.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)